templates/parachain/pallets/template/Cargo.toml

```diff
@@ -25,10 +25,13 @@ frame-benchmarking = { optional = true, workspace = true }
 frame-support = { workspace = true }
 frame-system = { workspace = true }
 
+# primitive deps
+sp-runtime = { workspace = true }
+sp-std = { workspace = true }
```

Comment:
Was `sp-std` be proposed to soft deprecate?
https://github.com/paritytech/polkadot-sdk/issues/2101

Reply:
hmm I see, a mistake indeed. Would you like to open a PR that reverts this + added a deprecation warning to sp-std, preventing this from happening?

Reply:
I think I can make a following PR to remove the usage of `sp-std` here.
I don't know how to warn deprecation for a crate...

Reply:
https://github.com/paritytech/polkadot-sdk/pull/5001
Following PR is here